manifests/p/ProxymanLLC/Proxyman/1.3.0/ProxymanLLC.Proxyman.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Automatically updated by the winget bot at 2023/Feb/19
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ProxymanLLC.Proxyman
+PackageVersion: 1.3.0
+PackageLocale: en-US
+Publisher: Proxyman LLC
+PackageName: Proxyman
+License: Copyright © 2022 Proxyman LLC. All rights reserved.

```

```suggestion
License: Proprietary
```
A copyright isn't a license